### PR TITLE
Require major.minor format for road_event_feed_info `version` field

### PR DESCRIPTION
**Pull request by j-d-b**
This PR resolves issue #80 by noting that the `version` field on the [road_event_feed_info](https://github.com/usdot-jpo-ode/jpo-wzdx/blob/master/feed-content/data-tables/road_event_feed_info.md) should follow the `major.minor` (e.g. `1.1`, `2.0` format).

~This will be required by the JSON schema (#79) once it is completed and merged in.~
Completed in e26ffa8

### CODE REVIEW

**Comment by sknick-iastate** [comment]
+1

**Comment by j-d-b** [comment]
Note #90 may require modifying this. We could however just not include the patch version here as nothing in the spec/output changes when the patch version changes.

**Comment by j-d-b** [comment]
I'm wondering if we should use this opportunity to also require this field, as it's currently optional.

File: create-feed/schemas/wzdx_v3.0_feed.json [hunk 1]
```diff
@@ -57,8 +57,9 @@
           "type": "string"
         },
         "version": {
-          "description": "The WZDx specification version used to create the data feed",
-          "type": "string"
+          "description": "The WZDx specification version used to create the data feed, in 'major.minor' format",
+          "type": "string",
+          "pattern": "^(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)$"
```
Comment (chuehlien) [inline create-feed/schemas/wzdx_v3.0_feed.json:62]: Can confirm that the pattern works well. 

Do we want to add an example in the description like what you've done for some other fields? Something like `(e.g. version 2.0 will be represented as '2.0')`. This is optional of course.

File: spec-content/data-tables/road_event_feed_info.md [hunk 2]
```diff
@@ -9,4 +9,4 @@ Field Name | Data Type | Description | Conformance | Notes
 feed_info_id |	ID |	Unique identifier for the data feed. |Required | Primary Key Recommendations on the format of this unique identifier will be made in the future. |
 feed_update_date |	DateTime |	The UTC date and time when the data feed was last updated. |	Required | All date/time formats shall use ISO 8601 Data elements and interchange formats – Information interchange. Example: `2016-11-03T19:37:00Z`
 metadata |	URL |	A link to the metadata file (WZ-metadata.txt). See [Metadata table](/spec-content/data-tables/metadata.md) for a description of the file. |	Optional |
-version |	Text |	The specification version used to create the data feed |	Optional	 |
+version |	Text |	The specification version used to create the data feed in `major.minor` format. |	Optional | Examples: `1.1`, `2.0`
```
Comment (chuehlien) [inline spec-content/data-tables/road_event_feed_info.md:12]: I noticed that you suggested making this field a "Required" field, which I agree is a good idea. Will that happen for v3?

Comment (j-d-b) [reply spec-content/data-tables/road_event_feed_info.md:12]: @Mahsa-Ettefagh @chuehlien  I missed this one but would definitely support making it required. The feed is a bit useless if it is not.

Comment (Mahsa-Ettefagh) [reply spec-content/data-tables/road_event_feed_info.md:12]: I agree with making this field required. This is the information that data producers can add themselves when setting up the feed and will be helpful to track feed versions in the future.

Comment (j-d-b) [reply spec-content/data-tables/road_event_feed_info.md]: Done. Note however this change is overwritten by the refactoring of the metadata/feed info which already made "version" required. Good to update here for reference though.

**Pull request merged**